tests/Kernel/fixtures/rendering.yml

```diff
@@ -2215,6 +2216,7 @@
           body: "Lorem Ipsum 6"
   assertions:
     count:
+      'h2.ecl-u-type-heading-2': 1
```

Comment:
I think it would be nice in another test to check that we don't have this h2 in case the title is empty.